inc/classes/optimization/CSS/class-critical-css.php

```diff
@@ -193,21 +193,22 @@ public function get_public_post_types() {
 				'fusion_template',
 				'blocks',
 				'jet-woo-builder',
-				'fl-builder-template'
+				'fl-builder-template',
 			]
 		);
 
 		$post_types = array_diff( $post_types, $excluded_post_types );
 		$post_types = esc_sql( $post_types );
 		$post_types = "'" . implode( "','", $post_types ) . "'";
 
-		$rows = $wpdb->get_results( // WPCS: unprepared SQL ok.
+		$rows = $wpdb->get_results(
 			"
 		    SELECT MAX(ID) as ID, post_type
 		    FROM (
 		        SELECT ID, post_type
 		        FROM $wpdb->posts
-		        WHERE post_type IN ( $post_types )
+				WHERE post_type IN ( $post_types )" .
+			"
```

Comment:
Why this concat?

Reply:
Fixed here 2cda9aa0fca1d3bd603e4a81bcccfb97bb5a72ce